### PR TITLE
chore: add example app

### PR DESCRIPTION
**Pull request by Datron**
## Problem
No example app for superposition

## Solution
bring in @knutties example app

### CODE REVIEW

**Comment by srid** [comment]
@Datron CI should succeed now. But please first merge https://github.com/juspay/superposition/pull/206

File: examples/superposition-demo-app/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Superposition Demo App setup instructions
+
+1. Compile and run the app
+```sh
+make superposition-example
+```
```
Comment (mahatoankitkumar) [inline examples/superposition-demo-app/README.md:6]: Shouldn't we have commands to run the sql file here and in the makefile?

**Comment by mahatoankitkumar** [comment]
Just a small nitpick, if we can add an override with hour_of_the_day as condition.

**Review by knutties — APPROVED** [review]

**Review by mahatoankitkumar — APPROVED** [review]

File: examples/superposition-demo-app/src/templates/index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="/static/htmx.js"></script>
```
Comment (ayushjain17) [inline examples/superposition-demo-app/src/templates/index.html:4]: why are we simply not using cdn provided by the packages, here ?
its anyways a network call over here, so why not fetch the original one, why make copies ?

Comment (knutties) [reply examples/superposition-demo-app/src/templates/index.html:4]: You can work on the project without internet 😉 @ayushjain17

File: docker-compose/postgres/db_init.sql
Comment (ShubhranshuSanjeev) [inline docker-compose/postgres/db_init.sql:1]: image
image

**Pull request merged**